pkg/restic/mocks/repository_manager.go

```diff
@@ -73,15 +74,15 @@ func (_m *RepositoryManager) InitRepo(repo *v1.BackupRepository) error {
 }
 
 // NewBackupper provides a mock function with given fields: _a0, _a1
-func (_m *RepositoryManager) NewBackupper(_a0 context.Context, _a1 *v1.Backup) (restic.Backupper, error) {
+func (_m *RepositoryManager) NewBackupper(_a0 context.Context, _a1 *v1.Backup) (uploader.Backupper, error) {
```

Comment:
I see we've moved the `NewBackupper` and `NewRestorer` out of repository_manager, so may also need to move the mocks?

Reply:
It's the mock for `RepositoryManager`, although we moved the `NewBackerupper` and `NewRestorer`, the `RepositoryManager` still needs to implement these two functions